checker/tests/lock/Issue2229.java

```diff
@@ -0,0 +1,6 @@
+import org.checkerframework.checker.lock.qual.*;
```

Comment:
One convention we're following is to have a link to the issue at the beginning of test files.
See for examples:
https://github.com/typetools/checker-framework/blob/master/checker/tests/lock/Issue805.java#L1

This is a bit redundant if the test file name contains the issue number, but is very helpful if a more descriptive name is used, e.g. in:

https://github.com/typetools/checker-framework/blob/master/checker/tests/lock/ThisSuper.java#L1

Please add such a link and, if you can think of a better file name, rename the test.

Reply:
I have made these modifications here: [https://github.com/t-rasmud/checker-framework/blob/issue2229/checker/tests/lock/DependentTypesNPE.java]( https://github.com/t-rasmud/checker-framework/blob/issue2229/checker/tests/lock/DependentTypesNPE.java)